test/expect-tests/dune_pkg_outdated/dune

```diff
@@ -0,0 +1,19 @@
+(library
+ (name dune_pkg_outdated_test)
+ (modules dune_pkg_outdated_test)
+ (inline_tests)
+ (libraries
+  dune_tests_common
+  stdune
+  dune_pkg_outdated
+  dune_console
+  threads.posix
```

Comment:
where do you use threads?

Reply:
I don't, I got confused it confused with the other transitive deps.